silk/config.py

```diff
@@ -2,15 +2,37 @@
 
 from django.utils import six
 
+from dealer.auto import auto
+
 from silk.singleton import Singleton
 
 
+
 def default_permissions(user):
     if user:
         return user.is_staff
     return False
 
 
+def default_revision():
+    revision = auto.revision
+    try:
+        # This is a pretty flaky way of getting revisions in a semi chronological
+        # order (of course two commits from the same second cannot be correctly
+        # ordered) for a couple of reasons:
+        #
+        #  - it only works for git
+        #  - it's using the protected _repo attribute from dealer
+        #  - the timestamp becomes visible in the revision
+        #
+        # However it is better than order commit hashes alphabetically
+        timestamp = auto._repo.git('show -s --format=%at ' + revision)
+        revision = ' # '.join([timestamp.decode(), revision])
+    except:
+        pass
```

Comment:
Does this make more sense to be `revision = ''`?

And does the rest of the app work correctly with an empty string or `None` value?  There seem to be several areas of the app that filter or group by revision value.

Reply:
@albertyw I removed the dealer dependency now - I think it's simpler just to let users to set the revision themselves using whichever method they prefer. This is what we have ended up doing when running this branch in production.